pkg/cmd/roachprod/main.go

```diff
@@ -1627,6 +1627,22 @@ var storageSnapshotCmd = &cobra.Command{
 	}),
 }
 
+var sideEyeRootCmd = &cobra.Command{
+	Use:   "side-eye",
+	Short: "interact with side-eye.io functionality",
+	Args:  cobra.MinimumNArgs(1),
+}
+var sideEyeSnapCmd = &cobra.Command{
+	Use:     "snapshot",
+	Aliases: []string{"snap"},
+	Short:   "interact with side-eye.io functionality",
```

Comment:
Could we make `Short` a little less vague and also add `Use` to describe what the CLI arg denotes. Incidentally, the above `storageSnapshotCmd` also takes a "snapshot", but of a very different kind.

Reply:
I've put more words in these command docs in the other PR, fwiw

Reply:
I pulled those extra words in here (word-for-word so it should still rebase cleanly)